docs/history.md

```diff
@@ -1,14 +1,20 @@
 # History
 
+## Development Version
+
+* ADD: Auto_Read function to read the data automatically using `xd.io.read` ({pull})`205` by [@syedhamidali](https://github.com/syedhamidali).
```

Comment:
This section should only include changes proposed in this pull request - please remove the other PRs notes

Reply:
Done!